src/components/Integrations/Table/ExpandedContent/PagerDutyExpandedContent.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListItemVariants,
+  TextListVariants,
+} from '@patternfly/react-core';
+import * as React from 'react';
+import { UserIntegrationType } from '../../../../types/Integration';
+import { getOuiaProps } from '../../../../utils/getOuiaProps';
+import {
+  ExpandedContentProps,
+  expandedContentTitleClass,
+} from '../ExpandedContent';
+
+export const PagerDutyExpandedContent: React.FunctionComponent<
+  ExpandedContentProps<UserIntegrationType>
+> = (props) => {
+  return (
+    <TextContent
+      {...getOuiaProps('Integrations/Table/PagerDutyExpandedContent', props)}
+    >
+      <TextList component={TextListVariants.dl}>
+        {'secretToken' in props.integration && (
+          <>
+            <TextListItem
+              className={expandedContentTitleClass}
+              component={TextListItemVariants.dt}
+            >
+              Integration Key
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {props.integration.secretToken !== undefined
+                ? 'Secret token'
```

Comment:
Do we want to just show text `"Secret token"`?

Reply:
Oh, no. It should just be the integration key and severity showing.

Reply:
This is currently how we are handling the other tokens in the integrations table - if it exists it shows string 'Secret Token'. We can follow up with another ticket, as mentioned, to replace that with '***xyz' and ability to show token or not.